ui/Screen/OrgRegistration.svelte

```diff
@@ -69,16 +70,19 @@
     // Start validating once the user enters something for the first time
     if (orgName && orgName.length > 0) validating = true;
     if (validating) validation.updateInput(orgName);
-
-    subject.name = orgName;
   }
 
   $: submitLabel =
     state === RegistrationFlowState.TransactionConfirmation
       ? "Submit transaction"
       : "Next";
-
   $: disableSubmit = $validation.status !== ValidationStatus.Success;
+
+  $: payer =
+    $session.status === Status.Success && getPayer($session.data.identity);
+  $: transaction = getTransaction(orgName);
```

Comment:
This is an oddly general method, esp. living in `org.ts`. What kind of transaction are we fetching here and from where?

Reply:
Maybe something like `generateTransaction` would make more sense - it's fetching the details of the potential transaction to display to the user so it's not quite at the stage of firing it off yet.

Reply:
But maybe it would fit better in `transaction.ts` with a named event (e.g. `getTransacton("Transactions.AddMemberToOrg")`

Reply:
What we want here is to look for a transaction specifically to a criteria as in type of message that it carries and what's inside it (org name). Seems likely utility `transaction.ts` should expose.